tests/test_integrations.py

```diff
@@ -82,4 +83,66 @@ def test_cross_framework_roundtrip():
     assert imported_tool.name == "addition"
 
     exported_smol = native_tool.export_to_smolagents()
-    assert exported_smol.forward(4, 5) == 9  
\ No newline at end of file
+    assert exported_smol.forward(4, 5) == 9
+
+
+# tests/test_integrations.py
+
+def test_bedrock_export(monkeypatch):
+    # Mock boto3 client
+    monkeypatch.setenv("AWS_ACCESS_KEY_ID", "test")
+    monkeypatch.setenv("AWS_SECRET_ACCESS_KEY", "test")
+
+    # Use a concrete tool instead of BaseTool
+    tool = Addition()
+    bedrock_config = tool.export_to_bedrock(lambda_arn="arn:aws:lambda:us-east-1:123456789012:function:test")
+
+    assert bedrock_config["toolName"] == "addition"
+    assert "num1" in bedrock_config["openAPISchema"]
+    assert bedrock_config["lambdaArn"] == "arn:aws:lambda:us-east-1:123456789012:function:test"
+
+def test_bedrock_import():
+    sample_tool = {
+        "toolName": "bedrock_addition",
+        "openAPISchema": json.dumps({
+            "openapi": "3.0.0",
+            "paths": {
+                "/add": {
+                    "post": {
+                        "description": "Add numbers",
+                        "requestBody": {
+                            "content": {
+                                "application/json": {
+                                    "schema": {
+                                        "type": "object",
+                                        "properties": {
+                                            "num1": {
+                                                "type": "number",
+
+                                            },
+                                            "num2": {
+                                                "type": "number",
+                                                
+                                            }
+                                        },
+                                        "required": ["num1", "num2"]
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }),
+        "lambdaArn": "arn:aws:lambda:us-east-1:123456789012:function:add"
+    }
+
+    # Use a concrete tool instead of BaseTool
+    tool = Addition()
```

Comment:
This isn't right. There should be a new tool created. Note on line 146. It's reading the description from the original `Addition` tool, not loading from the schema. I'm at EOD, but will fix tomorrow

Reply:
ie `tool = TestTool()` maybe a new tool that only exists to have imports grafted on to it

Reply:
This error is in import_from_bedrock, which might not be important anyway. (Would we ever import a tool from bedrock?)